README.md

```diff
@@ -37,14 +37,14 @@ has proved very successful.
 
 ## Process Overview
 
-The following process is used for targets in oss-fuzz:
+The following process is used for targets in OSS-Fuzz:
 
-- a target is accepted to oss-fuzz.
-- oss-fuzz build server build target fuzzers  regularly and submits them to
+- a target is accepted to OSS-Fuzz.
+- OSS-Fuzz build server build target fuzzers  regularly and submits them to
```

Comment:
OSS-Fuzz build server build**s**... + remove double space between "fuzzers" and "regularly"